specs/pvs/corla.pvs

```diff
@@ -1,33 +1,78 @@
+%%% found in the "Revised Draft of Proposed Rules" whose latest
+%%% version is labeled CCR 1505-1 and dated July 6, 2017. Section 25,
+%%% Post-Election Audit, of that document is the salient portion of
+%%% that document. We will call this document RDPR-6-Jul-2017 for the
+%%% purpose of traceability in this specification.
+%%%
+%%% This domain model is based upon that law, its rules, our contract,
+%%% our bibliography of publications about risk-limiting audits, etc.
+
+
+% @todo kiniry Add a discussion of our glossary and its relationship
+% to this specification.
+
+% @todo kiniry Define C.R.S. and explain the other sources used to
+% derive all concepts in this specification.
+
+%%% This document contains a formal specification of the RLA
+%%% tool.
+%%% This is a detailed technical specification of the tool,
 %%% annotated with prose designed to help readers who are not expert
 %%% in formal system specification understand the scope and interpretations
-%%% of the elements of the specification. The details of the specification
-%%% itself will guide deployment, management, maintenance and evolution
-%%% of the system delivered to the Colorado Department of State. The prose
-%%% annotations should help Colorado Department of State elections domain experts
+%%% of the elements of the specification. 
+%%% The specification documents our
+%%% thinking and the system design, from domain modeling and
+%%% engineering all of the way to formal specification and
+%%% verification.
+%%%
+%%% We write the formal specification using a
+%%% literate style, allowing the production of typeset books
+%%% and interactive hypertext directly from the specification.
+%%% The prose
+%%% annotations will help Colorado Department of State elections domain experts
 %%% and technical staff to assess the completeness and correctness of the
-%%% ColoradoRLA system/tool.
+%%% ColoradoRLA tool.
+%%% The details of the specification
+%%% will guide deployment, management, maintenance and evolution
+%%% of the system delivered to the Colorado Department of State. 
+%%%
+%%%\subsection{Technical Note}
+%% The RLA Tool is a client-server system. The server is implemented
+%%% in Java and runs on servers hosted by CDOS. The client is
+%%% implemented in TypeScript and JavaScript and runs in modern web
+%%% browsers.
+%%% CDOS provides
+%%% the hosting servers, network, and a number of services
+%%% in support of the tools deployment and management.
 %%%
 %%% This document is written in a literate style using the PVS
 %%% theorem prover. Writing system specifications in this fashion is
 %%% called (formal) domain engineering. Our formal domain model is
-%%% written in PVS's higher-order logic (HOL). Our informal domain
-%%% model is written in the Free \& Fair System Specification Language,
+%%% written in PVS's higher-order logic (HOL). There is a corresponding informal domain
+%%% model written in the Free \& Fair System Specification Language,
```

Comment:
Isn't that Free & Fair*_Election_ System Specification Language, here and a few other places? (quoting https://github.com/FreeAndFair/BON)?  Or are you re-thinking the name and positioning it as more general? Though the "E" in FAFSSL is handy for pronunciation. It could be "Free & Fair Extended System Specification Language I guess.